packages/components/organisms/f-checkout/src/components/Address.vue

```diff
@@ -13,7 +13,7 @@
                 label-style="inline"
                 is-grouped
                 :has-error="isAddressLine1Empty"
-                @input="updateAddressDetails('line1', $event)">
+                @input="updateFulfilmentDetails('Address', 'line1', $event)">
```

Comment:
Is there a way to show that `updateFulfilmentDetails ` is a method from `validations.mixin` somehow without searching through the whole project? It just a bit confusing when you try to understand where this method came from.

Reply:
I do agree, but it's partly the idea of the mixin to act as if the method was defined as part of the component. I think it's perhaps something we need to get used to, where if it's not listed in the file and isn't explicitly imported then it's probably from a mixin.

Reply:
I suppose one idea is prefixing the method names with `mixin_` or something

Reply:
Yeah, I quickly searched for the solution and basically it is what you said @xander-marjoram - we need to get used to it :grin: